examples/register.rs

```diff
@@ -1,16 +1,12 @@
-use nakama_rs::*;
 use nakama_rs::api_client::ApiClient;
 
 fn main() {
-    let mut client = ApiClient::new("defaultkey", "127.0.0.1", config::DEFAULT_PORT, "http");
+    let mut client = ApiClient::default();
```

Comment:
I guess in most real-world scenarios ApiClient will be created with custom server params
Also the `new` with all the params was a little closer to other nakama clients: https://heroiclabs.com/docs/authentication/

Maybe I am biased, but I got used that `default` is mostly used with some sort of options struct. Like, I would feel comfortable with something like `ApiClient::new(ConnectionOptions::default())`.

Reply:
Yes, that makes sense. I'll remove `default` for now